portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase

```diff
@@ -0,0 +1,163 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+    property testray.main.component.name = "Forms";
+
+    setUp {
+    TestCase.setUpPortalInstance();
+
+    User.firstLoginPG();
+
+    Navigator.openURL();
+    }
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			Navigator.openURL();
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Forms");
+
+			FormsAdmin.teardownForms();
+
+			Navigator.gotoNavItem(navItem = "Element Sets");
+
+			FormsAdmin.teardownElementSets();
+
+			PagesAdmin.tearDownCP();
+		}
+	}
+
+    @description = "Access Portal Global Menu and validate CONFIGURATION string at Control Panel"
+    @priority = "5"
+    test ValidateConfigurationStringAtControlPanel {
+        Click(locator1 = "//button[@data-qa-id='applicationsMenu']");
+        
+        Pause(locator1 = "3000");
+
+        Click(locator1 = "//span[contains(text(),'Control Panel')]");
+
+        Pause(locator1 = "3000");
+
+        AssertElementPresent(locator1 ="//h2[@id='nav_control_panel.configuration']");
```

Comment:
Why is the path present here in the testcase? Fix it.

Reply:
That's not important, i should had deleted this test.  Is that a problem?